socorro/lib/javautil.py

```diff
@@ -98,11 +98,7 @@ def parse_java_stack_trace(text):
                 stage = ADDITIONAL
 
         elif stage is ADDITIONAL:
-            # Verify this has a tab at the beginning of the line
-            if line[0] != "\t":
-                raise MalformedJavaStackTrace("additional line missing tab")
```

Comment:
I took out the "does it start with a tab?" bits. The result is that the additional section is closer to the original plus it doesn't enforce the "starts with a tab" thing which isn't always the case and is causing `MalformedJavaStackTrace` exceptions now.